docs/workflow/building/coreclr/README.md

```diff
@@ -1,56 +1,100 @@
-# Building
+# Building CoreCLR
 
-These are instructions for building the common form of CoreCLR on common platforms. For instructions on particular form factors, see also [Build NativeAOT](nativeaot.md), [Build CoreCLR on Linux](linux-instructions.md), [Build CoreCLR on OS X](osx-instructions.md), [Build CoreCLR on FreeBSD](freebsd-instructions.md),
-[Cross Compilation for ARM on Windows](cross-building.md), [Cross Compilation for Android on Linux](android.md).
+* [Introduction](#introduction)
+* [Common Building Options](#common-building-options)
+  * [Build Drivers](#build-drivers)
+  * [Extra Flags](#extra-flags)
+  * [Build Results Layout](#build-results-layout)
+* [Platform-Specific Instructions](#platform-specific-instructions)
+* [Testing CoreCLR](#testing-coreclr)
 
-To build just CoreCLR, use the `-subset` flag to the `build.sh` (or `build.cmd`) script at the repo root:
+## Introduction
 
-For Linux:
-```
-./build.sh -subset clr
+Here is a brief overview on how to build the common form of CoreCLR in general. For further specific instructions on each platform, we have links to instructions later on in [Platform-Specific Instructions](#platform-specific-instructions).
+
+To build just CoreCLR, use the `subset` flag to the `build.sh` or `build.cmd` script at the repo root:
+
+For Linux and MacOS:
+
+```bash
+./build.sh --subset clr
```

Comment:
Dtto - <code>./build.sh clr</code> works just fine for me.

Reply:
I see this is explained in more detail at the end of the section. I think it should actually go in the front to explain how the command line is structured.